### ISSUE TITLE
setuptools is too old

### ISSUE THREAD

**Issue by corydodt** [corydodt/circus-base#6]
```
Installing collected packages: setuptools
  Found existing installation: setuptools 20.7.0
    Uninstalling setuptools-20.7.0:
      Successfully uninstalled setuptools-20.7.0
Successfully installed setuptools-41.0.0
```

Versions of setuptools prior to version 31 run into this bug: https://github.com/pypa/setuptools/issues/885

Update setuptools in the base image.

**Comment by corydodt** [comment]
This is fixed in the release-0.6 codeline

**Comment by corydodt** [comment]
fixed
